db/models/assessment.py

```diff
@@ -0,0 +1,64 @@
+import uuid
+
+from db import db
+from sqlalchemy.exc import IntegrityError
+
+
+class Assessment(db.Model):
+    id = db.Column(
+        "id",
+        db.Text(),
+        default=lambda: str(uuid.uuid4()),
+        primary_key=True,
+    )
+    application_id = db.Column(db.Text(), index=True, unique=True)
+
+    def __repr__(self):
+        return "<Assessment {} for Application {}>".format(
```

Comment:
🤔 Typically reprs mirror how an object instantiated, but that could be complicated here...something to think about. Lovely use of a dunder method though 💯 (f string instead of format....? :P)

Reply:
Obv f string - thanks, done